aplib.net-demo/Assets/Scripts/Inventory/Inventory.cs

```diff
@@ -4,7 +4,7 @@
 using UnityEngine;
 using UnityEngine.UI;
 
-[RequireComponent(typeof(AmmoPouch))]
+[RequireComponent(typeof(RawImage))]
```

Comment:
What is this?

Reply:
The ammo pouch was moved to the player/enemies, but the inventory breaks when it doesn't have a RawImage, so I added this